mozilla/foundation.mozilla.org
# tailwind classes for buttons, updated js

Closes #8037

**Steps to test**:

1. Visit https://foundation-s-share-butt-p0ashv.herokuapp.com/en/youtube/regretsreporter/
2.  Click on the share button
3. Other share buttons should animate/hide as expected
4. If code and animation look ok, testing is complete!
